cmd/docs/README.md

```diff
@@ -1,13 +1,14 @@
 # WSO2 Micro Integrator Command Line tool
 
 You can view and manage the Micro Integrator instances using the “mi” command line tool. Some of the usages of the command line tool include,
-1. Get a list of deployment runtime artifacts.
+1. Get a list of deployed artifacts in the Micro Integrator runtime.
 2. Inspect details of each runtime artifact such as a proxy service or an API.
 3. Get the invocation endpoint of an artifact.
+4. Update log levels and download log files.
```

Comment:
Shall we add View and manager users in an external user store?

Reply:
Added